service/raft/raft_address_map.hh

```diff
@@ -139,17 +139,17 @@ class raft_address_map {
         // an expiration period (the time period since the last access lies within
         // the given expiration period time frame).
         bool expired(clock_duration expiry_period) const {
-            auto last_access_delta = Clock::now() - *_ptr->_last_accessed;
+            auto last_access_delta = Clock::now() - *_ptr._last_accessed;
             return expiry_period < last_access_delta;
         }
 
-        timestamped_entry* timestamped_entry_ptr() {
+        struct timestamped_entry& timestamped_entry() {
             return _ptr;
         }
 
     private:
         list_type& _expiring_list;
-        timestamped_entry* _ptr;
+        struct timestamped_entry& _ptr;
```

Comment:
Why do you need `struct`?

Reply:
Name conflict with `timestamped_entry()` function.
This field gets removed in a later commit anyway